openedx/core/djangolib/testing/tests/test_utils.py

```diff
@@ -27,3 +28,27 @@ def test_user_is_attached_to_mock_request(self):
     def test_mock_request_without_user(self):
         request = get_mock_request()
         self.assertIsInstance(request.user, AnonymousUser)
+
+
+class TestToggleSwitch(TestCase):
+    """
+    Verify that the toggle_switch utility can be used to turn Waffle Switches
+    on and off.
+    """
+    def test_toggle_switch(self):
+        """Verify that a new switch can be turned on and off."""
+        name = 'foo'
+
+        switch = toggle_switch(name)
+
+        # Verify that the switch was saved.
+        self.assertEqual(switch, Switch.objects.get())
+
+        # Verify that the switch has the right name and is active.
+        self.assertEqual(switch.name, name)
+        self.assertTrue(switch.active)
+
+        switch = toggle_switch(name, active=False)
```

Comment:
Similar to above comment, we're checking that it returns the value from the db, and we're checking explicit 'False', should we check explicit 'True' as well?  I realize that the default is current True, but should that change...